### DISCUSSION TITLE
Migrate to C++17

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#1935] [General]
changing [deprecated](https://en.cppreference.com/w/cpp/error/uncaught_exception) `std::uncaught_exception` into `std::uncaught_exceptions` in just two files seems enough to compile successfully in C++17.
- json.cpp
- sdltiles.cpp

**Comment by Coolthulhu** [comment]
There's some mac issue with the `exceptions`. It complains about compatibility.

**Comment by scarf005** [comment]
resolved in #1953 .
